src/apps/frontend/contexts/auth.provider.tsx

```diff
@@ -18,6 +18,14 @@ type AuthContextType = {
   loginError: AsyncError;
   loginResult: AccessToken;
   logout: () => void;
+  sendOTP: (
+    countryCode: string,
+    phoneNumber: string,
+  ) => Promise<AuthRecord>;
```

Comment:
When you send OTP, system will not return anything, just like signup. The return type is incorrect.